plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/TrinoS3FileSystem.java

```diff
@@ -722,6 +723,7 @@ private boolean deleteObject(String key)
             return true;
         }
         catch (AmazonClientException e) {
+            log.debug(e, "Failed to delete object '%s' from the bucket %s", key, bucketName);
```

Comment:
I think it should be a warn level. cc @losipiuk 

BTW the return value of this method is ignored. Can you make it void and rename to `tryDeleteObject`?

BTW why do we ignore deletion failures in `delete(Path path, boolean recursive)`, but not so in `deleteFiles(Collection<Path> paths)`? 
(no change requested for now)

Reply:
Actually - why log instead propagating exception up the stack? Should the decision if deletion failure can be ignored or not be done but code up the stack?

Reply:
this is used to delete some additional marker objects, maybe that's why.

cc @electrum